.github/workflows/ci.yml

```diff
@@ -13,8 +13,14 @@ jobs:
       uses: actions/setup-ruby@v1
       with:
         ruby-version: 2.6.x
```

Comment:
Does this mean that it is only tested with ruby 2.6?  If we don't have any ruby version conditional code, then we should test with the minimum supported (2.4) and latest released (2.7).

Reply:
Oh, this is only being used for rubocop at the moment.  Although, that makes me wonder why we are installing database clients (libmysqlclient-dev libpq-dev) if we don't have a step/job to run the test suite.

Reply:
We are still using travis for test runs because orchestrating a MySQL DB for testing with github actions isn't super straightforward. I thought I'd defer working on a script to do that, and run a matrix of tests/versions for later. The client packages are just so we can `gem install`.